pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -22,6 +22,7 @@
     <!-- Exclude shims from the closure verification -->
     <ExcludeFromClosure Include="mscorlib" />
     <ExcludeFromClosure Include="System" />
+    <ExcludeFromClosure Include="System.ComponentModel.Composition" />
```

Comment:
Why is this needed?

Reply:
@weshaggard I made this because I thought it was the correct resolution to the build errors we were getting about cyclical dependencies?

Reply:
No we shouldn't do that. In fact this should be needed as it is no longer inbox after we removed the shim build. 

Do note that removing the shim from inbox is technically a breaking change but it only type forwarded one type (`Lazy<T,M>`) so I don't think we really need to worry or care about that as it likely isn't even used by anyone and if it was then people can reference the new package we are going to create for this library.

Reply:
OK @maryamariyan sounds like you should revert this edit and the other one in Microsoft.Private.CoreFx.UAP.pkgproj